src/backing/client/client.h

```diff
@@ -37,11 +37,11 @@ class Client {
   // `GetPublicKey`.
   virtual StatusOr<std::string> AsymmetricSign(
       std::string key_version_resource_id, DigestCase digest_case,
-      std::string digest_bytes) = 0;
+      std::string digest_bytes) const = 0;
```

Comment:
Are changes to this file meant to be part of this PR?

Reply:
Yes---added to allow for EngineData to return a const reference to Client and then callers to call methods on that client:

```
engine_data()->client().GetPublicKey
```